### ISSUE TITLE
Remove names from OGOR Reporting Instructions CSV and ASCII

### ISSUE THREAD

**Issue by jennmalcolm** [ONRR/onrr.gov-site#230]
See shared email box to have requester review. 

Please see attachments for OGOR Reporting Instructions CSV and ASCII - https://www.onrr.gov/ReportPay/production-reporting.htm - under Sample Layouts.  Christina Garcia and Denise McGee need to be removed as contacts. Denise retired, Christina went to another department.  

[https://app.zenhub.com/files/210892300/4bcf7f80-3d48-487b-89e5-329d705823c1/download](https://app.zenhub.com/files/210892300/4bcf7f80-3d48-487b-89e5-329d705823c1/download)

[https://app.zenhub.com/files/210892300/52b962ce-129d-49a3-85c3-66ff1daeafb4/download](https://app.zenhub.com/files/210892300/52b962ce-129d-49a3-85c3-66ff1daeafb4/download)

**Comment by jennmalcolm** [comment]
@lpgoldstein the requester sent a message to our shared box that the CSV file wasn't updated (see below), so I'm reopening.


I noticed on the ONRR.Gov website that the ASCII Record Layout was updated, but the CSV Record Layout was not updated.  Snippets are below.  I sent attachments for both ASCII and CSV.  Let me know if I need to go through GitHub.  I appreciate your help.
![image](https://user-images.githubusercontent.com/30806839/87719774-eb438200-c781-11ea-8dc3-ff4ba99c333c.png)

![image](https://user-images.githubusercontent.com/30806839/87719824-00201580-c782-11ea-80dd-063dcdaedcd0.png)


ASCII Record Layout -
